### PR TITLE
Spec for AA linkage with grafana

### PR DESCRIPTION
**Pull request by a2batic**
tendrl-bug-id: #191

### CODE REVIEW

**Comment by a2batic** [comment]
@brainfunked @anivargi @gnehapk @cloudbehl Please review

**Comment by cloudbehl** [comment]
@a2batic please add bug-id in the commit message.

**Comment by a2batic** [comment]
@cloudbehl added now

File: specs/authentication_and_authorisation_linkage_be.adoc [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+## Problem Description
+
+Since the Tendrl stack is going to include Grafana for monitoring, so there will be a button in UI of Tendrl which can take users to Grafana Dashboard to view different charts.Tendrl has an authentication system and Grafana has there own authentication system as well. So instead of having two authentication for users, we are planning to bring it down to one.
+  
+## Use Cases
+Provide seemless authorization between two dashboards.
+There are three types of users in Tendrl:- 
+* Admin
+* Limited
+* Viewer
```
Comment (gnehapk) [inline specs/authentication_and_authorisation_linkage_be.adoc]: @a2batic I think types of users are - Admin, Limited and Normal. Refer API doc - https://github.com/Tendrl/api/blob/master/docs/authorizaton.adoc

Comment (a2batic) [reply specs/authentication_and_authorisation_linkage_be.adoc]: @gnehapk, yes, I will correct this

File: specs/authentication_and_authorisation_linkage_be.adoc [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+  - Users authenticated via tendrl should be automatically authenticated in grafana as well.
+  - Launching a cluster specific grafana dashboard should be pre-authorised from within tendrl ui.
+
+
+## Problem Description
+
+Since the Tendrl stack is going to include Grafana for monitoring, so there will be a button in UI of Tendrl which can take users to Grafana Dashboard to view different charts.Tendrl has an authentication system and Grafana has there own authentication system as well. So instead of having two authentication for users, we are planning to bring it down to one.
+  
+## Use Cases
+Provide seemless authorization between two dashboards.
```
Comment (shtripat) [inline specs/authentication_and_authorisation_linkage_be.adoc:16]: s/seemless/seamless/

File: specs/authentication_and_authorisation_linkage_be.adoc [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+## Proposed change
+
+The Authentication seamlessness by enabling anonymous login
+```
+[auth.anonymous]
+enabled = true
+org_name = YourOrgName
+org_role = Viewer/Editor/Read-only editor
+```
+For every user there will be an organization created in grafana
```
Comment (shtripat) [inline specs/authentication_and_authorisation_linkage_be.adoc:45]: Based on recent discussions in daily meetings, there needs to be some changes I feel here. An org for every user is the point I am emphasizing on.

Comment (a2batic) [reply specs/authentication_and_authorisation_linkage_be.adoc:45]: Yes, I need to update this

File: specs/authentication_and_authorisation_linkage_be.adoc (hunk 2 above)
Comment (a2batic) [reply specs/authentication_and_authorisation_linkage_be.adoc:16]: Acknowledge

**Pull request closed without merging**